modules/bitgo/src/v2/wallet.ts

```diff
@@ -2135,12 +2135,15 @@ export class Wallet implements IWallet {
     // this could return 100 build transactions
     const buildResponse = (await this.bitgo
       .post(this.baseCoin.url('/wallet/' + this.id() + '/consolidateAccount/build'))
-      .send(whitelistedParams)
-      .result()) as any;
+      .send(whitelistedParams)) as any;
 
     // we need to step over each prebuild now - should be in an array in the body
     const consolidations: PrebuildTransactionResult[] = [];
-    for (const consolidateAccountBuild of buildResponse) {
+    if (Object.keys(buildResponse.body).length === 0 && buildResponse.status === 204) {
```

Comment:
What "Object.keys(buildResponse.body).length === 0" mean? Isn't better if you extract this logic to a constant?